tracer/src/Datadog.Trace/ClrProfiler/Instrumentation.cs

```diff
@@ -329,6 +329,9 @@ internal static void InitializeNoNativeParts(Stopwatch sw = null)
                 {
                     Log.Debug("Initializing activity listener.");
                     Activity.ActivityListener.Initialize();
+
+                    Log.Debug("Initializing OpenTelemetry components.");
+                    OpenTelemetry.Sdk.Initialize();
```

Comment:
I named this `Datadog.Trace.OpenTelemetry.Sdk` to better map the concepts to the OpenTelemetry SDK namespacing/file structure, but I'm open to feedback!

Reply:
In the `catch` below, the error message it now slightly off:
```
Log.Error(ex, "Error initializing activity listener");
```

Which makes me thing, should we call these two `Initialize()` methods in separate `try`/`catch` clauses? Should a failing in the first one stop the second one from running?

Reply:
Good call, so far the OpenTelemetry SDK fix I have should be operating independently of our ActivityListener so I'll break that out into its own try/catch 👍🏼

Reply:
I've separated the try/catch clauses in b60809fbc6ecd061ba37c912ed1dbb25a76fe674

Reply:
Thanks!